src/matrixHelpers/FormioLoader/index.js

```diff
@@ -0,0 +1,202 @@
+const initFormioInstance = (formioElem, opts) => {
+  // if already initiated, reject
+  if (formioElem.dataset.formioFormUrl) return;
+  // if doesn't have required options, reject
+  if (!opts.envUrl || !opts.projectName || !opts.formName) {
+    console.warn(
+      "Require envUrl, projectName, formName to initiate the form.",
+      opts
+    );
+    return;
+  }
+  const bodyContainer = $("body");
+  const defaultRedirect = "contact-us/response/";
```

Comment:
this seems very dangerous to have as a default.

Reply:
didn't think about this as it is existing logic, seems like it's only for form that has `applicationSubmit ` binded (line101-118)...
tbh i don't quite understand where is the use case for this custom block (probably need to find out which forms fire this event in the formio platform), i just try to maintain the backward compatibility atm